shared/signup/phone-number/phone-input.tsx

```diff
@@ -376,7 +378,10 @@ class _PhoneInput extends React.Component<Kb.PropsWithOverlay<Props>, State> {
 
   render() {
     return (
-      <Kb.Box2 direction={isMobile ? 'vertical' : 'horizontal'} style={styles.container}>
+      <Kb.Box2
+        direction={isMobile ? 'vertical' : 'horizontal'}
+        style={Styles.collapseStyles([styles.container, !isMobile && this.state.focused && styles.highlight])}
```

Comment:
may be worth extracting `!isMobile && this.state.focused` into a separate boolean; I could see it being used in more places if this component grows. Also it makes the logic a little cleaner imo.

Reply:
Discussed in chat and leaving as is.